rebasehelper/cli.py

```diff
@@ -240,6 +248,17 @@ def __getattr__(self, name):
 class CliHelper(object):
 
     @staticmethod
+    def set_colors(app):
```

Comment:
I think it would be more logical to first check value of `app.conf.colors`, and only if it's `auto`, then continue with checking other things. I also think it'd be nice to honor `PY_COLORS` environment variable.